src/openmcmc/mcmc.py

```diff
@@ -32,7 +32,8 @@ class MCMC:
                      specified will be sampler from prior distributions
         samplers (list): list of the samplers to be used for each parameter to be estimated.
         n_burn (int): number of initial burn in these iterations are not stored.
-        n_iter (int): number of iterations which are stored in store.
+        n_thin (int): number of iterations to thin by.
+        n_iter (int): number of iterations which are stored in store. For every iteration, n_thin iterations are run.

```

Comment:
Thanks- it's clear from the comment how this works. It might also be a good idea to add a bit to the text part of the docstring, to indicate clearly to the user that `(self.n_burn + self.n_iter) * self.n_thin` iterations will get run in total, as I think people won't spot this at first glance. (So in this instance, if I just changed `self.n_thin` to `10`, I would get `100000` iterations using the other defaults).